Add release notes for Kyma 1.24 Ekibastuz (#744)
kyma-project/website: content/blog-posts/2021-06-17-release-notes-1.24/index.md

---
title: "Kyma 1.24 Ekibastuz"
author:
  name: "Karolina Zydek, Technical Writer @Kyma"
tags:
  - release-notes
type: release
releaseTag: "1.24.0"
redirectFrom:
  - "/blog/release-notes-124"
---

> With engines on and all hands on deck, we are fully concentrated on the upcoming major release of Kyma 2.0. For this reason, this release is rather a short stop on a longer journey, meant to apply all patches that would allow us to continue a secure journey. Read on to find out what we had to fix in this release. 

Reviewer: `journey` in this intro appears twice relatively closely. Would you like to change it to something else? 
Like:
```suggestion
With engines on and all hands on deck, we are fully concentrated on the upcoming major release of Kyma 2.0. For this reason, this release is rather a short stop on a longer journey, meant to apply all patches that would allow us to continue in a safe and secure manner. Read on to find out what we had to fix in this release. 
```
or `to safely continue`
or `on a longer route`

I also wonder about the official straightforward mention of Kyma 2.0.

Reviewer 2: In the [previous release notes](https://kyma-project.io/blog/2021/6/2/release-notes-123/), we already mentioned Kyma 2.0 so I guess it won't hurt to mention it once more 🤷‍♀️

<!-- overview -->

See the overview of all changes in this release:

> - [Installation](#installation) - Istio upgraded to 1.10.0, updated ORY charts

Reviewer: ```suggestion
- [Installation](#installation) - Istio upgraded to 1.10.0, ORY charts updated
```

This is nitpicking :)

Author: Ok!

## Installation 

### Istio upgraded to 1.10.0 

In this release, we upgraded Istio from 1.9.5 to 1.10.0. 

For more details on the introduced changes, read the official [Istio 1.10.0 release notes](https://istio.io/latest/news/releases/1.10.x/announcing-1.10/).  

> ### Updated ORY charts 

Reviewer: ```suggestion
### ORY charts updated
```

Again, nitpicking :)

We also updated ORY charts, changing the Oathkeeper version to 0.38.11-beta.1. 

> For more details on the introduced changes, read the official [Oathkeeper 0.38-11-beta.1 release notes](https://github.com/ory/oathkeeper/releases/tag/v0.38.11-beta.1) 

Reviewer: ```suggestion
For more details on the introduced changes, read the official [Oathkeeper 0.38-11-beta.1 release notes](https://github.com/ory/oathkeeper/releases/tag/v0.38.11-beta.1).
```

---
title: "Kyma 1.24 Ekibastuz"
author:
  name: "Karolina Zydek, Technical Writer @Kyma"
tags:
  - release-notes
type: release
releaseTag: "1.24.0"
redirectFrom:
  - "/blog/release-notes-124"
---

With engines on and all hands on deck, we are fully concentrated on the upcoming major release of Kyma 2.0. For this reason, this release is rather a short stop on a longer route, meant to apply all patches that would allow us to continue a secure journey. Read on to find out what we had to fix in this release. 

<!-- overview -->

See the overview of all changes in this release:

> - [Installation](#installation) - Istio upgraded to 1.10.0, ORY charts updated

Reviewer: ```suggestion
- [Installation](#installation) - Istio upgraded to 1.10.0, ORY charts updated
- [Serverless](#serverless) - Python 3.8 deprecated and new Python 3.9 runtime introduced
```

## Installation 

### Istio upgraded to 1.10.0 

In this release, we upgraded Istio from 1.9.5 to 1.10.0. 

For more details on the introduced changes, read the official [Istio 1.10.0 release notes](https://istio.io/latest/news/releases/1.10.x/announcing-1.10/).  

### ORY charts updated

We also updated ORY charts, changing the Oathkeeper version to 0.38.11-beta.1. 

For more details on the introduced changes, read the official [Oathkeeper 0.38-11-beta.1 release notes](https://github.com/ory/oathkeeper/releases/tag/v0.38.11-beta.1). 

## Serverless 

### Introducing Python 3.9 and deprecating 3.8

> In this release we are intruducing a Python 3.9 serverless runtime. This is the latest feature release of Python and it is the recommended version of Python that gets the highest attention from Python's community and early security patches.

Reviewer: ```suggestion
In this release, we introduced the Python 3.9 Serverless runtime, which is the latest feature release of Python. It is the recommended Python version, and it gets the highest attention from the community and early security patches.
Following this recommendation, we are deprecating our Python 3.8-based runtime as we plan to remove Python 3.8 from the list of supported runtimes **in the next release**. That's why we recommend that you configure all your Python 3.8 Functions to run on the Python 3.9 runtime.
```
